### DISCUSSION TITLE
Spectrum analyzer for Hydrogen audio

### DISCUSSION THREAD

**Discussion by bobesponja93** [hydrogen-music/hydrogen#1934] [Hydrogen on Linux]
I tried downloading ladspa spectrogram plugins and inserting them into the strips, but Hydrogen only shows plugin controls, not the plugin ui/graph. So how do you use spectrum analyzers with Hydrogen?

**Comment by trebmuh** [comment]
If you're running Linux, you can go through JACK and use a spectrogram outside Hydrogen to monitor what's coming out of Hydrogen.

**Comment by theGreatWhiteShark** [comment]
Hey @bobesponja93 ,

As @trebmuh pointed out you can easily use a standalone analyzer in combination with the JACK driver of Hydrogen. I can recommend `jaaa` and `japa` (see http://kokkinizita.linuxaudio.org/linuxaudio/).

AFAIK the LADSPA specification does not support graphical element, which was one of the main drivers to develop its successor LV2. In the latter plugin type, you can have arbitrary graphical elements (as you need for a spectrogram). But this is not supported (yet) by Hydrogen.
